src/components/toast/toast.style.ts

```diff
@@ -49,6 +51,26 @@ const StyledPortal = styled(Portal)<{
       bottom: 0;
       top: auto;
       width: 100%;
+    `}    
+
+    ${alignY === "top" &&
+    css`
+      top: 0;
+      bottom: auto;
+    `}
+
+    ${alignY === "center" &&
+    css`
+      top: 50%;
+      transform: translate(${align === "left" ? "50%" : "-50%"}, -50%);
+    `}
+
+    ${alignY === "bottom" &&
+    css`
+      bottom: 0;
+      top: auto;
+      display: flex;
```

Comment:
Question: Do we need flex here?

Reply:
I do think `flex` needs to be added here for `flex-direction` to take effect - this is mainly for stacked toasts to appear in bottom to top order when `alignY` is set to "bottom". I could remove this if its preferred that stacked toasts have the same order from top to bottom in all cases.